pkgs/tools/admin/google-cloud-sdk/default.nix

```diff
@@ -13,7 +13,7 @@
 let
   pythonInputs = [ cffi cryptography pyopenssl crcmod ]
                  ++ lib.optional (with-gce) google-compute-engine;
-  pythonPath = lib.makeSearchPath python.sitePackages pythonInputs;
+  pythonEnv = python.withPackages (_: pythonInputs);
```

Comment:
This is OK for the application case, but not when using as library.

Reply:
But this isn't a python library. Am I missing something?

This is just a bundle of CLI utils as far as I can tell.

Reply:
yet it is in `python-modules.nix`. If it is not usable as a library it should not be here.

Reply:
Alright. I removed it from there and changed the expression to just take an abitrary `python` argument that will be used to construct the required environment.